.circleci/config.yml

```diff
@@ -760,6 +760,12 @@ jobs:
             if [[ $CIRCLE_NODE_INDEX == "6" ]]; then export CYPRESS_RECORD_KEY="$CYPRESS_RECORD_KEY"; fi
             if [[ $CIRCLE_NODE_INDEX == "6" ]]; then yarn run cy:run:e2e-ci -- --record --key $CYPRESS_RECORD_KEY --ci-build-id "$CIRCLE_BUILD_NUM-$CIRCLE_BRANCH" --group "e2e bug tests" --spec "cypress/integration/04-bug-tests/*.cy.js"; fi
   
+      - run:
+          name: report coverage for e2e
+          when: always
```

Comment:
does this mean it gets run on every commit?

Reply:
I think so? right @thetif ?
I believe that's what we have with the other front-end tests too

Reply:
I have to admit I'm not completely sure what always means, but I think it might change if you only wanted it to to that step on failures or successes. But like Darren said, the other reporting steps do the same thing, so I was just following it.